custom_components/sat/climate.py

```
@@ -154,9 +157,7 @@ def __init__(self, coordinator: SatDataUpdateCoordinator, config_entry: ConfigEn
         self._attr_name = str(config_entry.data.get(CONF_NAME))
         self._attr_id = str(config_entry.data.get(CONF_NAME)).lower()
 
-        self.thermostat = config_entry.data.get(CONF_THERMOSTAT)
-        self._climates = config_entry.data.get(CONF_SECONDARY_CLIMATES) or []
-        self._main_climates = config_entry.data.get(CONF_MAIN_CLIMATES) or []
+        self._radiators = config_entry.data.get(CONF_RADIATORS) or []
```

Ruff (0.8.2)

`CONF_RADIATORS` may be undefined, or defined from star imports

(F405)